### PR TITLE
7658 add petitioner modals

### PR DESCRIPTION
**Pull request by codyseibert**
Screen Shot 2021-02-15 at 1 10 31 PM

### CODE REVIEW

File: web-client/src/presenter/sequences/openMatchingEmailFoundModalSequence.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { stopShowValidationAction } from '../actions/stopShowValidationAction';
+
+export const openMatchingEmailFoundModalSequence = showProgressSequenceDecorator(
+  [
+    stopShowValidationAction,
+    clearAlertsAction,
+    checkEmailAvailabilityAction,
+    {
+      emailAvailable: [setShowModalFactoryAction('MatchingEmailFoundModal')],
+      emailInUse: [setShowModalFactoryAction('NoMatchingEmailFoundModal')],
```
Comment (matthopson) [inline web-client/src/presenter/sequences/openMatchingEmailFoundModalSequence.js]: Are these backwards?

File: web-client/src/views/CaseDetail/PetitionerInformation.jsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { EditPrivatePractitionersModal } from './EditPrivatePractitionersModal';
 import { EditSecondaryContactModal } from '../EditSecondaryContactModal';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { FormGroup } from '../../ustc-ui/FormGroup/FormGroup';
+import { MatchingEmailFoundModal } from './MatchingEmailFoundModal';
 import { PractitionerExistsModal } from './PractitionerExistsModal';
```
Comment (matthopson) [inline web-client/src/views/CaseDetail/PetitionerInformation.jsx]: Add NoMatching... modal?

**Review by matthopson — APPROVED** [review]
Just a couple small things (that may not matter at this time)

File: web-client/src/presenter/sequences/openMatchingEmailFoundModalSequence.js (hunk 1 above)
Comment (codyseibert) [reply web-client/src/presenter/sequences/openMatchingEmailFoundModalSequence.js]: yes lol

File: web-client/src/views/CaseDetail/PetitionerInformation.jsx (hunk 2 above)
Comment (codyseibert) [reply web-client/src/views/CaseDetail/PetitionerInformation.jsx]: this was unused, needed to delete this line

**Pull request merged**